Modules/Finance/SubModules/Export.cs

```diff
@@ -0,0 +1,14 @@
+﻿using ConnectWise.Http.ModuleTypes;
+
+namespace ConnectWise.Http.Modules.Finance.SubModules
+{
+	public class ExportSubModule: PartialSubModule
+	{
+		internal ExportSubModule(string module, string endpoint) : base(module, endpoint) { }
+
+		public CWRequest ExportRequest(object exportAccountingBatchRequest)
```

Comment:
I would prefer if this endpoint took in the already serialized `string` as well instead of an `object`.

Also please add some comments to the endpoint!